app/controllers/message_threads_controller.rb

```diff
@@ -35,7 +35,9 @@ def index
     if params[:filter_id].present?
       @filter = policy_scope(Filter, policy_scope_class: FilterPolicy::ScopeShowable).find_by(id: params[:filter_id])
     else
-      @filter = Current.user.visible_filters.first
+      if Current.user.visible_filters.any?
```

Comment:
Toto sa mi uplne nepozdava, ze tu davame redirect. v podstate sa nebude dat linkovat root (vzdy to spravi redirect). Nedava skor zmysel nastavit ten filter a nerobit redirect?

Reply:
Ok, prerobil